spk/mitmproxy/src/installer.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+# Package
+PACKAGE="mitmproxy"
+DNAME="mitmproxy"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="mitmproxy"
+GROUP="users"
```

Comment:
I belive the `users` group isn't correct here. Does the package need file system access? For DSM5, we  generally use the `nobody` group instead.

FTR, DSM6 users will not be member of the `users` group at all (we'll just use our `sc-*` groups if access to the FS is needed), and one thing to check if if there's even a `nobody` group.